rplugin/python3/discord/__init__.py

```
@@ -55,7 +55,11 @@ def update_presence(self):
                 return
             self.discord = Discord(client_id, reconnect_threshold)
             with handle_lock(self):
-                self.discord.connect()
+                try:
+                    self.discord.connect()
+                except:
```

we should not silently eat up unrelated exceptions